### ISSUE TITLE
Hardcoded status icons

### ISSUE THREAD

**Issue by varlesh** [clementine-player/Clementine#5413]
Status icons (Play/Pause) on tab and system tray hardcoded, this spoil style.

![screenshot_20160619_101818](https://cloud.githubusercontent.com/assets/8083855/16175911/66f9be7a-3607-11e6-804e-7604237460fd.png)

**Comment by FadeMind** [comment]
FYI https://github.com/clementine-player/Clementine/search?utf8=%E2%9C%93&q=media-playback-start.png

**Comment by varlesh** [comment]
@FadeMind Thx. Need png?
```
src/ui/qtsystemtrayicon.cpp
QIcon icon(":icon.png");
```

**Comment by varlesh** [comment]
I convert media-playback-start.svg to media-playback-start.png - this not working.
Also i found "tiny-start", but this not working too: 
`QIcon tiny_start = IconLoader::Load("tiny-start", IconLoader::Other);`
https://github.com/clementine-player/Clementine/blob/master/src/ui/systemtrayicon.cpp

**Comment by FadeMind** [comment]
```
 <file>tiny-pause.png</file>
<file>tiny-start.png</file>
```
https://github.com/clementine-player/Clementine/tree/ccfce616b839913719d328172199fb59678a7b63/data

https://github.com/clementine-player/Clementine/blob/ccfce616b839913719d328172199fb59678a7b63/data/tiny-pause.png
https://github.com/clementine-player/Clementine/blob/ccfce616b839913719d328172199fb59678a7b63/data/tiny-start.png

**Comment by varlesh** [comment]
hardcoded

**Comment by FadeMind** [comment]
@hatstand please look at this in a free time. Thanks.

**Comment by narunlifescience** [comment]
this is no more hard coded...  I fixed this in #5210. you should build from the git repository to use it.. you can customize all the icons by following https://github.com/narunlifescience/Clementine-Custom-Icon-Sets

**Comment by varlesh** [comment]
Thx
